main.py

```diff
@@ -323,6 +323,74 @@ async def clear_scores() -> None:
 
     print(f"Deleted all clearable scores in {time.time() - start_time:.2f} seconds")
 
+CHAMPION_BADGE_ID = 67
+FORMER_CHAMPION_BADGE_ID = 66
+
+async def update_champion_badges() -> None:
+    for rx in (0, 1, 2):
+        if rx == 0:
+            redis_board = "leaderboard"
+            modes = ("std", "taiko", "ctb", "mania")
+        elif rx == 1:
+            redis_board = "relaxboard"
+            modes = ("std", "taiko", "ctb")
+        else:  # rx == 2:
+            redis_board = "autoboard"
+            modes = ("std",)
+
+        for mode in modes:
+            mode_int = mode + (4 * rx)
+            rank_key = f"ripple:{redis_board}:{mode}"
+
+            # this is updated in a cron so we need to also check current rank one
+            users_who_peaked_at_rank_one = list(
```

Comment:
jesus it's still using cmyui-pkg

Reply:
probably worth prioritizing moving to a better pkg, i don't think it's easy to do transactions w/ `cmyui-pkg` & this is scary to me w/o them lol

Reply:
this entire code-base is bad and unstructured, the whole thing needs a redo lol